ads/_config.js

```diff
@@ -1017,6 +1017,8 @@ const adConfig = jsonConfiguration({
     renderStartImplemented: true,
   },
 
+  'vadserver': {},
```

Comment:
I see you are not implementing `[renderStart](https://github.com/ampproject/amphtml/blob/47d8f204b1546b0ab7eff07be6757a198c68538d/ads/README.md#available-apis)`. We recommend using this API if possible, as it will give you some performance gains, but it is optional.